.gitignore

```diff
@@ -2,3 +2,6 @@
 /.lintcache
 /node_modules
 npm-debug.log
+
+# Ignore sublime project files.
+*.sublime-*
```

Comment:
I would prefer to keep that outside, there are tens of different dev configurations, and I wouldn't like to favorize any single one here. So we should either contain ignores for all, or for none. I prefer second option.

You can easily clear those files from a status by using global .gitignore options in your environement